integration-tests/__tests__/helpers.js

```diff
@@ -81,26 +81,25 @@ export const forceClick = async (_app, _selector) => {
   }, _selector);
 };
 
-const getImageSnaphotConfig = async (app, options = {}) =>
+const getImageSnaphotConfig = async app =>
   app.client.execute(() => window.devicePixelRatio)
     .then(({ value: devicePixelRatio }) => ({
       ...defaultImageSnaphotConfig,
       customSnapshotIdentifier: ({ testPath, currentTestName, counter }) =>
         `${devicePixelRatio}x-${appSize}-`
           .concat(kebabCase(`${path.basename(testPath)}-${currentTestName}-${counter}`)),
-      ...options,
     }));
 
-export const matchImageSnapshot = async (app, options = {}) => {
+export const matchImageSnapshot = async (app, rect = null) => {
```

Comment:
Because options wasn't being used?

Reply:
Yeah, and I didn't see the need for an interface for it here.